src/index.css

```diff
@@ -0,0 +1,198 @@
+/* green: #677767 */
+/* beige: #F7F6F1 */
+/* white: #fff */
+/* black: #000 */
+
+html, body {
+    font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+}
```

Comment:
Add `width, height: 100%` and `justify-content: center` to the body, because when the authentication form is displayed, it is not in the center.